armData.json

```diff
@@ -19254,6 +19254,86 @@
         "imageURL": "./imgs/1040416100.png",
         "en": "[4★] Nereus Staff"
     },
+    "聖布の槍": {
+        "ja": "聖布の槍",
+        "element": "light",
+        "type": "spear",
+        "skill1": "normalSanteL",
+        "skill2": "non",
+        "element2": "light",
+        "skill3": "non",
+        "element3": "light",
+        "minhp": 47,
+        "minattack": 421,
+        "hp": 290,
+        "attack": 2550,
+        "series": "none",
+        "slvmax": 10,
+        "maxlv": 100,
+        "imageURL": "./imgs/1040213400.png",
+        "en": "Sacred Standard"
+    },
+    "[4凸]聖布の槍": {
+        "ja": "[4凸]聖布の槍",
+        "element": "light",
+        "type": "spear",
+        "skill1": "normalSanteL",
+        "skill2": "opus-normalElement",
+        "element2": "light",
+        "skill3": "non",
+        "element3": "light",
+        "minhp": 47,
+        "minattack": 421,
+        "hp": 351,
+        "attack": 3082,
+        "series": "none",
+        "slvmax": 15,
+        "maxlv": 150,
+        "hplv100": 290,
+        "attacklv100": 2550,
+        "imageURL": "./imgs/1040213400.png",
+        "en": "[4★] Sacred Standard"
+    },
+    "アヤの弓": {
+        "ja": "アヤの弓",
+        "element": "light",
+        "type": "bow",
+        "skill1": "normalKatsumiM",
+        "skill2": "non",
+        "element2": "light",
+        "skill3": "non",
+        "element3": "light",
+        "minhp": 38,
+        "minattack": 417,
+        "hp": 208,
+        "attack": 2600,
+        "series": "none",
+        "slvmax": 10,
+        "maxlv": 100,
+        "imageURL": "./imgs/1040709400.png",
+        "en": "Iridescent Bow"
+    },
+    "[4凸]アヤの弓": {
+        "ja": "[4凸]アヤの弓",
+        "element": "light",
+        "type": "bow",
+        "skill1": "normalKatsumiM",
+        "skill2": "non",
+        "element2": "light",
+        "skill3": "non",
+        "element3": "light",
+        "minhp": 38,
+        "minattack": 417,
+        "hp": 269,
+        "attack": 3142,
```

Comment:
>         "hp": 269,
>         "attack": 3142,

Is this status true? I could not find reliable information.
([4★] Iridescent Bow)